docker-hub/api/latest.yaml

```
@@ -18,37 +19,89 @@ info:
     In addition to the [Docker Hub UI](https://docs.docker.com/docker-hub/) and [Docker Hub CLI tool](https://github.com/docker/hub-tool#readme) (currently experimental),
     Docker provides an API that allows you to interact with Docker Hub.
     Browse through the Docker Hub API documentation to explore the supported endpoints.
+
 tags:
-  # Primary objects
-  - name: Authentication
+  - name: rate-limiting
+    x-displayName: Rate Limiting
+    description: |
+      The Docker Hub API is limited on the amount of requests you can perform
+      per minute against it.
+
+      If you haven't hit the limit, each request to the API will return the
+      following headers in the response.
+      - `X-RateLimit-Limit` - The limit of requests per minute.
+      - `X-RateLimit-Remaining` - The remaining amount of calls within the limit
+      period.
+      - `X-RateLimit-Reset` - The unix timestamp of when the remaining resets.
+
+      If you have hit the limit, you will receive a response status of `429` and
+      the `X-Retry-After` header in the response.
+
+      The `X-Retry-After` header is a unix timestamp of when you can call the
+      API again.
```

```suggestion
      API again.
      
      
      Note: These rate limits are separate from anti-abuse and Docker Hub download, or pull, rate limiting. To learn more about Docker Hub pull rate limiting, see [Docker Hub Download rate limit](https://docs.docker.com/docker-hub/download-rate-limit/)
```

You might want to put a snippet in there to explicitly call out the difference between the API rate limiting. 

Come to think of it, I wonder if it would be goo to link from /docker-hub/download-rate-limit back to this page 🤔 . It might be helpful so that you can easily navigate between the rate limiting documentation we have.

---

Good call. Yeah would be good to link as well as differentiate the different rate-limiting we have.

---

Added the suggestion. Thanks!